packages/client/components/Insights.tsx

```diff
@@ -34,7 +38,8 @@ const Insights = (props: Props) => {
     {UNSTABLE_renderPolicy: 'full'}
   )
   const {viewer} = data
-  const {domains} = viewer
+  const {domains, company} = viewer
+  useSegmentTrack('Viewed domain stats', {domainId: company?.id})
```

Comment:
-1 this will fail if the person's email address doesn't match the domain they're viewing (ie the consultant scenario)

instead, issue 1 call for every viewer.domains.id